utils/api_uploader.py

```diff
@@ -1,52 +1,115 @@
+    Anime - Raw                          : "1_4"
+Audio
+    Lossless                             : "2_1"
+    Lossy                                : "2_2"
+Literature
+    Literature - English-translated      : "3_1"
+    Literature - Non-English             : "3_2"
+    Literature - Non-English-Translated  : "3_3"
+    Literature - Raw                     : "3_4"
+Live Action
+    Live Action - English-translated     : "4_1"
+    Live Action - Idol/Promotional Video : "4_2"
+    Live Action - Non-English-translated : "4_3"
+    Live Action - Raw                    : "4_4"
+Pictures
+    Pictures - Graphics                  : "5_1"
+    Pictures - Photos                    : "5_2"
+Software
+    Software - Applications              : "6_1"
+    Software - Games                     : "6_2"
+"""
+# ################################# CATEGORIES MUST BE EXACT ##################################
+
+# ###################################### EXAMPLE REQUEST ######################################
+"""
 # Required
-torrent_name = ""
+file_name = "/path/to/my_file.torrent"
+# Required
+category = "6_1"
+# Required
+display_name = "API upload example"
+
+# May be blank
+information = "API HOWTO"
+# May be blank
+description = "Visit #nyaa-dev@irc.rizon.net"
+# Default is 'n' No
+# Change to 'y' Yes to set
+is_anonymous : 'n',
+is_hidden    : 'n',
+is_remake    : 'n',
+is_complete  : 'n'
+"""
+# #############################################################################################
 
+# ######################################## CHANGE HERE ########################################
+# Required
+file_name = ""
 # Required
-main_cat = ""
+category = ""
 # Required
-sub_cat = ""
+display_name = ""
 
 # May be blank
 information = ""
 # May be blank
 description = ""
-# flags = [Hidden, Remake, Complete, Anonymous]
-# 0 for NOT SET / 1 for SET
-# Required
-flags = [0, 0, 0, 0]
+# Default is 'n' No
+# Change to 'y' Yes to set
+is_anonymous = 'n'
+is_hidden    = 'n'
+is_remake    = 'n'
+is_complete  = 'n'
+# #############################################################################################
 
-metadata={
-    "username": username,
-    "password": password,
-    "display_name": torrent_name,
-    "main_cat": main_cat,
-    "sub_cat": sub_cat,
-    "information": information,
-    "description": description,
-    "flags": flags
-    }
+# #################################### DO NOT CHANGE BELOW ####################################
+# ############################ UNLESS YOU KNOW WHAT YOU ARE DOING #############################
+auth_info = {
+    "username"     : username,
+    "password"     : password
+}
 
-# Required
-file_name = ""
+metadata={
+    "category"     : category,
+    "display_name" : display_name,
+    "information"  : information,
+    "description"  : description,
+    "is_anonymous" : is_anonymous,
+    "is_hidden"    : is_hidden,
+    "is_remake"    : is_remake,
+    "is_complete"  : is_complete
+}
 
 files = {
```

Comment:
Do we have to send everything as files...? There's no problem in sending variables and files at the same time in `requests.post(data=data)`.